doc/releases.md

```diff
@@ -1,8 +1,38 @@
 # Releases
 
-See [the HoloViz blog](https://blog.holoviz.org/tag/hvplot.html) for a visual summary of the major features added in a selection of releases.
+See [the HoloViz blog](https://blog.holoviz.org/#category=hvplot) for a visual summary of the major features added in a selection of releases.
 
-## Version 0.9.0
+## Version 0.9.1 (2023-12-22)
```

Comment:
I have no opinion about adding the release dates to the header. I just wanted to say this will show up in the right sidebar:

![image](https://github.com/holoviz/hvplot/assets/19758978/31e022c2-445c-431f-bfce-3e9c4c0f9ceb) 


Maybe having this as a `###` with `## Version 0.9` above it (This is what is done in [HoloViews](https://holoviews.org/releases.html).)

Reply:
I copied Datashader :) https://datashader.org/releases.html It'd be nice one day to unify how all the releases pages look across HoloViz. For instance, Panel no longer has it part of the main toctree.